src/commands/gate/README.md

```diff
@@ -1 +1,62 @@
-WORK IN PROGRESS
\ No newline at end of file
+# GATE command
+
+Evaluate Quality Gates rules in Datadog.
+
+## Usage
+
+### Commands
+
+#### `evaluate`
+
+The `evaluate` command evaluates the matching Quality Gates rules in Datadog.
+
+```bash
+datadog-ci gate evaluate [--scope] [--tags] [--dry-run] [--fail-if-unavailable] [--fail-on-empty]
+```
+
+For example:
+
+```bash
+datadog-ci gate evaluate --scope team:backend --scope team:frontend --fail-on-empty
+```
+
+- `--scope` is an array of key value pairs of the form `key:value`. This parameter sets additional scope when retrieving matching rules. Only the rules matching the scope provided will be evaluated.
+- `--tags` is an array of key value pairs of the form `key:value`. This parameter sets global tags that are applied to all results. The upload process merges the tags passed on the command line with the tags in the `DD_TAGS` environment variable. If a key appears in both `--tags` and `DD_TAGS`, the value in `DD_TAGS` takes precedence.
+- `--dry-run` runs the command without the final evaluation step. All other checks are performed. The default value is `false`.
+- `--fail-if-unavailable` fails the command if Datadog is unavailable. The default value is `false`.
+- `--fail-on-empty` fails the command if no matching rules are found in Datadog. The default value is `false`.
+
+### Environment variables
+
+Additionally, you can configure the `gate` command with the following environment variables:
+
+- `DATADOG_API_KEY` or `DD_API_KEY` (**required**): The API key used to authenticate the requests.
+- `DATADOG_APP_KEY` (**required**): The application key used to authenticate the requests.
+- `DD_TAGS`: Sets global tags applied to all spans. The format must be `key1:value1,key2:value2`. The upload process merges the tags passed on the command line with the tags in the `--tags` parameter. If a key appears in both `--tags` and `DD_TAGS`, the value in `DD_TAGS` takes precedence.
```

Comment:
unrelated to the changes, but would we want a `DD_SCOPE`?

Reply:
Mmm do you think it will be useful for clients? What's the main reason we have `DD_TAGS` and `--tags`?

Reply:
ergonomics: sometimes it might be easier to set an env var than it is to modify the command. No big difference though

Reply:
Makes sense, noted it down for future 👍